docs/src/apimanual.md

```diff
@@ -1132,6 +1132,7 @@ const MOIB = MOI.Bridges
 import FooBar
 const optimizer = FooBar.Optimizer()
 MOI.set(optimizer, MOI.Silent(), true)
+MOI.set(optimizer, MOI.TimeLimit(), 0)
```

Comment:
Adding this line here means that all the tests will be executed with a `TimeLimit` of 0 hence they will all fail

Reply:
Just read the doc. Disabling the time limit for the tests may be annoying as you might have your tests looping infinitely